kufar/books/main.go

```diff
@@ -166,96 +182,99 @@ func (d Database) SignUpUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Write([]byte(http.StatusText(http.StatusOK)))
+	w.Write([]byte(http.StatusText(http.StatusOK) + ": auth passed"))
 }
 
-//SignInUser ...
-func (d Database) SignInUser(w http.ResponseWriter, r *http.Request) {
-	credReq := CredentialsRequest{}
-	if err := json.NewDecoder(r.Body).Decode(&credReq); err != nil {
-		http.Error(w, http.StatusText(http.StatusInternalServerError)+": error unmarshalling json", http.StatusInternalServerError)
-		return
-	}
+func (d Database) getBookUserID(idInt int) (int, error) {
+	row := d.db.QueryRow("select userid from books where id = $1", idInt)
 
-	row := d.db.QueryRow("select uuid, password from credentials where username = $1", credReq.Username)
+	var userid int
```

Comment:
`userid` it's not a Go convention. In Go, we are using camel case, so it will be `userID`.
[doc](https://golang.org/doc/effective_go.html#names)